src/librustdoc/doctest/extracted.rs

```diff
@@ -0,0 +1,132 @@
+use serde::Serialize;
```

Comment:
There should be a doc-comment here explaining what feature this is supporting, and that `FORMAT_VERSION` needs to be bumped when the types are changed.

Reply:
Very good point!